src/fides/api/alembic/migrations/versions/fc2b2c06e595_add_detection_discovery_tables.py

```diff
@@ -131,7 +131,7 @@ def downgrade():
 
     # Staged resource table
     op.drop_index(op.f("ix_stagedresource_resource_type"), table_name="stagedresource")
-    op.drop_index(op.f("ix_stagedresource_key"), table_name="stagedresource")
+    op.drop_index(op.f("ix_stagedresource_urn"), table_name="stagedresource")
```

Comment:
unrelated to main point of the PR, but i noticed this when trying to downgrade, i'd been sloppy in keeping this downgrade consistent with the actual table definition, so it broke the downgrade command. fixing this up.